src/applications/edu-benefits/10215/sass/10215-edu-benefits.scss

```diff
@@ -0,0 +1,7 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-process-list";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-form-process";
+@import "../../../../platform/forms/sass/m-schemaform";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
+@import "../../../../platform/forms/sass/m-form-confirmation";
```

Comment:
Same here, could you please update these import paths from `/formation/sass/...` to `/css-library/dist/stylesheets/...` for all module imports?